app/controllers/doctors_controller.rb

```diff
@@ -1,12 +1,27 @@
 class DoctorsController < ApplicationController
+    include StatesHelper
+    include HTTParty
+  def find
+      @states = helpers.states
+    if search_params[:first_name] != "" && search_params[:last_name] != ""
+      @our_doctors = Doctor.where(first_name: search_params[:first_name], last_name: search_params[:last_name])
```

Comment:
we talked about this -- think it's fine now that our internal DB is small to show all doctors by that name, but it doesn't make a lot of sense to me why we would search our internal db differently than our API. May just be able to implement @our_doctors = Doctor.where(search_params) 
(or at the very least @our_doctors = Doctor.where(doctor_args) )